build.gradle

```diff
@@ -123,6 +122,10 @@ dependencies {
     testImplementation deps.testContainers.toxiproxy
     testImplementation deps.pgsql
     testImplementation deps.awaitility
+    testImplementation deps.exposed
+    testImplementation deps.koin.core
+    testImplementation deps.koin.coreExt
+    testImplementation deps.joda
```

Comment:
joda time is now only used in the tests to illustrate how to integrate with exposed etc.